app/authorization.go

```diff
@@ -252,3 +252,12 @@ func (a *App) SessionHasPermissionToManageBot(session model.Session, botUserId s
 
 	return nil
 }
+
+func (a *App) SessionHasPermissionToSendAlerts(session model.Session) bool {
```

Comment:
Can we directly use a.SessionHasPermissionTo as SessionHasPermissionToSendAlerts is doing almost the same

Reply:
this wrapper method will be helpful to extend any logic specific to Alerts in future